BR: refine BR docs (#8628)
pingcap/docs: br/backup-and-restore-design.md

---
title: BR Design Principles
summary: Learn about the design details of BR.
---

# BR Design Principles

This document describes the design principles of BR, including its architecture<!--, design philosophy,//--> and backup files.

## BR architecture

BR sends a backup or restoration command to each TiKV node. After receiving the command, TiKV performs the corresponding backup or restoration operation.

> Each TiKV node has a path in which the backup files generated in the backup operation are stored and from which the stored backup files are read during the restoration.

Reviewer: can we make this statement simpler

> Each TiKV node has a path in which the backup files generated in the backup operation are stored and from which the stored backup files are read during the restoration.

Reviewer: ```suggestion
Each TiKV node will access the storage path during backup and  restoration.
```

Author: Agree with this modification. Maybe we should synchronize the Chinese as well.

![br-arch](/media/br-arch.png)

<!--## BR design philosophy

For details about the BR design philosophy, see [TBD](xxx).//-->

## Backup files

This section describes the design of backup files generated by BR.

### Types of backup files

BR can generate the following types of backup files:

- `SST` file: stores the data that the TiKV node backs up.

> - `backupmeta` file: stores the metadata of this backup operation, including the number, the key range, the size, and the Hash (sha256) value of the backup files.

Reviewer: ```suggestion
- `backupmeta` file: stores the metadata of this backup, including the number, the key range, the size, and the Hash (sha256) value of the backup files.
```
